### DISCUSSION TITLE
MSW fails to start

### DISCUSSION THREAD

**Discussion by zoharlevin** [mswjs/msw#1536] [Q&A] [answered]
i have a react app that uses msw with deferred mounting.  i am trying to upgrade from v 0.36.8 to v 1.0.1 and the worker.start() doesn't end.  i debugged it and saw that what happens is exactly as described [here](https://github.com/mswjs/msw/discussions/1011) - except that i'm not on safari, i'm on chrome.  and i do not have any calls to `serviceWorker.unregister()` anywhere in my app.

however i still get stuck on the call to `requestIntegrityCheck` which never returns.  
any help would be much appreciated 🙏

**Comment by zoharlevin** [comment, marked as answer]
oops, sorry, my bad... it was solved once i upgraded `mockServiceWorker.js` 🤦‍♀️ 
you can delete this discussion

**Reply by kettanaito** [reply]
Hey, @zoharlevin. No worries. I will keep it there just in case somebody else would be looking and stumble upon your solution, fixing it for them. You never know how many people may have exactly the same issue as you.

**Reply by cstephens-cni** [reply]
@zoharlevin how did you upgrade it,  I redid my init and can see it using the new mockServiceWorker however mine still hangs?

**Reply by zoharlevin** [reply]
@cstephens-cni  i don't remember exactly but i'm pretty sure all i did was `init` and a new file was generated with the correct version (you can see the version number in the file)
